### PR TITLE
Fixed JS strings in templates.

### PR DESCRIPTION
**Pull request by Mimino666**

### CODE REVIEW

**Comment by FinalAngel** [comment]
nice work @Mimino666 can you also mention this in the changelog?

File: cms/templates/cms/toolbar/toolbar_javascript.html [hunk 1]
```diff
@@ -115,7 +115,7 @@
         'id': '{{ request.toolbar.clipboard.pk|unlocalize }}',
         'url': '{% if request.toolbar.clipboard.pk %}{% cms_admin_url "cms_page_clear_placeholder" request.toolbar.clipboard.pk %}{% endif %}'
     },
-    'messages': '{% filter escapejs %}{% if messages %}{% for message in messages %}{{ message }}{% endfor %}{% endif %}{% endfilter %}',
+    'messages': '{% filter escapejs %}{% if messages %}{% for message in messages %}{{ message|escapejs }}{% endfor %}{% endif %}{% endfilter %}',
```
Comment (vxsx) [inline cms/templates/cms/toolbar/toolbar_javascript.html]: this one is already escaped, no?

Comment (Mimino666) [inline cms/templates/cms/toolbar/toolbar_javascript.html]: You are right. Good catch!
When I discovered the error, I was working with the older version of CMS and didn't notice it was already fixed in latest branch.

**Comment by vxsx** [comment]
Thanks a lot for the contribution, could you actually rebase this onto `release/3.4.x` branch?

**Comment by czpython** [comment]
Thanks @Mimino666.
Please reopen against the release/3.4.x branch.

**Pull request closed without merging**